.ado/windows-vs-pr.yml

```diff
@@ -695,7 +685,6 @@ jobs:
       - Setup
```

Comment:
We might as well skip this entire step. The only reason it is here is for the testing of the nuget package...

Reply:
I was wanting to keep it here since it's the only place where we get coverage for packing desktop. Not sure how valuable the coverage actually is though, but we've seen packing issues in publish before when we didn't test in PR.

Reply:
Talked offline, but the small amount of coverage here probably isn't worth it. Removing the job.